tests/periph_flashpage/main.c

```diff
@@ -283,8 +283,8 @@ static const shell_command_t shell_commands[] = {
     { "info", "Show information about pages", cmd_info },
     { "dump", "Dump the selected page to STDOUT", cmd_dump },
     { "dump_local", "Dump the local page buffer to STDOUT", cmd_dump_local },
-    { "read", "Read and output the given page", cmd_read },
-    { "write", "Write (ASCII) data to the given page", cmd_write },
+    { "read", "Copy the given page to the local page and dump to STDOUT", cmd_read },
```

Comment:
I'd say there's no local "page" but ram "buffer"... I suggest:
`Load the given page into RAM and dump it to STDOUT`.